python/activity_stream/activity_stream.py

```diff
@@ -158,6 +159,30 @@ def note_widget(self):
         """
         return self.ui.note_widget
 
+    def _get_clickable_user_icons(self):
+        """
+        Whether user icons in the activity stream display as clickable.
```

Comment:
Can we also mention the default value (e.g. hand)?
What happens when you get the arrow? Are mouse clicks ignored?